src/w2form.js

```diff
@@ -1032,8 +1032,6 @@
                     case 'email':
                     case 'pass':
                     case 'password':
-                        field.el.value = value;
-                        break;
```

Comment:
Above fields do not have options, therefore there is no point for it to have options object.